src/js/cookieMessage.js

```diff
@@ -2,6 +2,45 @@ class CookieMessage {
 
 	constructor (CookieMessageEl) {
 		this.CookieMessageEl = CookieMessageEl;
+
+		return this.setup();
+	}
+
+	setup () {
+		const message = document.querySelector('.o-cookie-message');
+
+		if (userHasConsentedToCookies()) {
+			hideMessage();
+			return;
+		}
+
+		setupMessage();
+
+		function setupMessage () {
```

Comment:
it feels like all these functions should go outside of the setup function and be part of the class